src/sentry/static/sentry/app/redux/reducers/processAssignment.js

```diff
@@ -0,0 +1,122 @@
+import {t} from 'app/locale';
+import IndicatorStore from 'app/stores/indicatorStore';
+
+import {
+  PROCESS_ASSIGN_SELECT_PRESET,
+  PROCESS_ASSIGN_SELECT_PROCESS_DEF,
+  PROCESS_ASSIGN_SET_VARIABLE,
+  PROCESS_ASSIGNMENTS_POST_REQUEST,
+  PROCESS_ASSIGNMENTS_POST_SUCCESS,
+  PROCESS_ASSIGNMENTS_POST_FAILURE,
+  PROCESS_ASSIGNMENTS_SET_EDITING,
+} from '../actions/processAssignment';
+
+export const initialState = {
+  errorMessage: null,
+  processById: {},
+  indicatorToken: null,
+
+  // assignment
+  assignPresetId: null,
+  assignProcessDefinitionId: null,
+  assignVariables: {},
+
+  editing: false,
+};
+
+const processAssignment = (state = initialState, action) => {
+  switch (action.type) {
+    case PROCESS_ASSIGN_SELECT_PRESET:
```

Comment:
In future, perhaps we could have a discussion whether this kind of state changes is necessary to have in action/reducers? It is a very simple state change, without api calls. Is the currently selected preset used in any other form outside of the preset form?

Reply:
It's a good question. Actually the state of this preset stuff wasn't really that simple and I really liked using redux for it. I then simplified it because of a bug! But when it was complex, redux really shone.

In a nutshell it had to do with the different state that changes - if you select a preset, then the process and variables change, if you selected the process and variables, the preset changed (which was actually something I simplified later) and there were some corner cases there, which with a state framework like redux, I gotta say got very elegant.

The question about whether something should reside in redux or not is perhaps a hard one, I feel that it might be difficult to set clear rules about it. Let's discuss it later!